### ISSUE TITLE
missing code

### ISSUE THREAD

**Issue by meli-lewis** [meli-lewis/goodreads#2]
Emily: I'm feeling crazy. Was there code for stacking in the notebook you saw? I'm not seeing it in what I submitted, but I have it in a .py file locally.

**Comment by emmmbrown** [comment]
Nope, I don't think so... there were a couple calls to a t4 that I had no idea where they came from, so I changed them to t2 since that's what it looked like you intended. I definitely thought something got left out.

**Comment by meli-lewis** [comment]
how bizarre! I added it again. $20 says I had my notebook open in more than
one tab and lost track, ugh.

**Comment by emmmbrown** [comment]
It happens. Especially when you're too lazy to separate websites and notebooks into different windows, which is my problem...
